src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -792,15 +795,23 @@ public override void VisitBinaryOperatorExpression(IBinaryOperatorExpression ope
         {
             LogString(nameof(IBinaryOperatorExpression));
 
-            var kindStr = $"{nameof(BinaryOperationKind)}.{operation.BinaryOperationKind}";
+            var kindStr = $"{nameof(BinaryOperatorKind)}.{operation.OperatorKind}";
             if (operation.IsLifted)
             {
-                LogString($" ({kindStr}-IsLifted)");
+                kindStr += "-IsLifted";
```

Comment:
To be more consistent with other IOperation tests, we should probably have these as separate fields, with true/false explicitly listed.